tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature

```diff
@@ -27,20 +27,20 @@ Feature: restrict Sharing
     And user "Brian" has uploaded file "lorem.txt" to "simple-folder/lorem.txt" in the server
     And user "Brian" has logged in using the webUI
 
-  @smokeTest
+  @smokeTest @skipOnOCIS
   Scenario: Restrict users to only share with users in their groups
     Given the setting "shareapi_only_share_with_group_members" of app "core" has been set to "yes" in the server
+#   shares only with groups members is not available now in the oCIS webUI
```

Comment:
Maybe this comment should be above the scenario the below step is for opening the share dialog which technically should be possible in ocis

Reply:
I'm asking this because when I saw the comment I thought it was for the step below 😕

Reply:
I am saying  it doesn't matter because I thought i clearly mentioned **shares only with groups members** which is connected with above step only

Reply:
The comment in between the scenario is misleading IMHO too.